app/presenters/guest_presenter.rb

```diff
@@ -18,4 +18,8 @@ def win_header_partial
   def nav_drawer_submenu_partial
     "components/guest_nav_drawer_submenu"
   end
+
+  def welcome_message_partial
```

Comment:
Not really for this PR, but I wonder if we should think about some other organizational layer that collects these partials in a single directory. I suppose this is what the ViewModels are supposed to do. Should we move GuestPresenter to a GuestViewModel in a future PR?